interfaces/builtin/i2c.go

```diff
@@ -17,8 +17,23 @@
  *
  */
 
-package boot
+package builtin
 
-var (
-	ImportAssertionsFromSeed = importAssertionsFromSeed
+import (
+	"github.com/snapcore/snapd/interfaces"
 )
+
+const i2cConnectedPlugAppArmor = `
+/dev/i2c-[0-9]* rw,
+`
```

Comment:
I think `i2c` should probably be renamed to `i2c-control` if this stays as is.

However, I'm not sure of the direction of this PR. It seems like this should be more like hidraw and serial-port where these devices is the udev backend along with slot attributes provided by gadget yaml to expose the slots to consuming apps. Is there a reason you did not pursue this route?

CC: @niemeyer and @morphis